data/scripts.js

```diff
@@ -3644,9 +3644,20 @@ exports.BattleScripts = {
 		var pool = Object.keys(sets).randomize();
 		var ranks = {'~':'admins', '&':'leaders', '@':'mods', '%':'drivers', '+':'voices'};
 		var levels = {'~':99, '&':97, '@':96, '%':96, '+':95};
-		for (var i = 0; i < 6; i++) {
-			var rank = pool[i].charAt(0);
+		var count = 0;
+		var i = -1;
```

Comment:
why two variables?? both `i` and `count` have the same value inside the loop body, b/w lines 3651 and 3673